src/client/search/CurrentQueryTerms.js

```diff
@@ -1,56 +1,30 @@
 import React, { Component } from 'react';
-import { connect } from 'react-redux';
-import { addSearchTerm } from './SearchActions';
 import SearchTermsList from './SearchTermsList';
+import AddSearchTerm from './AddSearchTerm';
 
 let nextSearchTermId = 0;
-let CurrentQueryTerms = ({ dispatch }) => {
-  let showSearchkeyword = false;
-  let searchKeyword;
+
+const CurrentQueryTerms = () => {
+  const showAddSearchTerm = () => {
+    //searchTermContainer and addSearchTerm are in ./AddSearchTerm
+    if ($('#searchTermContainer').is(':hidden')) {
```

Comment:
Please add to the comment that you're aware this is a bad practise, but is done for the sake for simplicity and ease of coding